jupyter_server/base/handlers.py

```diff
@@ -195,6 +193,10 @@ def login_available(self):
     def authorizer(self):
         return self.settings["authorizer"]
 
+    @property
+    def identity_provider(self):
+        return self.settings["identity_provider"]
```

Comment:
Comparing this to the "Authorizer" (#713), should this field on the tornado settings be optional?